ambry-account/src/main/java/com/github/ambry/account/mysql/DatasetDao.java

```diff
@@ -110,74 +111,80 @@ public DatasetDao(MySqlDataAccessor dataAccessor, MySqlAccountServiceConfig mySq
             DATASET_TABLE, ACCOUNT_ID, CONTAINER_ID, DATASET_NAME, VERSION_SCHEMA, LAST_MODIFIED_TIME, RETENTION_POLICY,
             RETENTION_COUNT, RETENTION_TIME_IN_SECONDS, USER_TAGS);
     updateDatasetIfExpiredSql = String.format(
```

Comment:
Do you mind add one line comment for each sql prepared statement?

Reply:
Sure, I'll add the comment for each sql.